fc/jira/triage_task.py

```diff
@@ -75,10 +73,14 @@ class TriageTask(Task):
     def from_json(cls, json: dict, auth: Auth):
         new_task = cls()
         super(TriageTask, new_task).from_json(json, auth)
-        # next 3 instance variables will be populated with next set of changes, for now None
-        new_task.importance = None
-        new_task.level_of_effort = None
-        new_task.due_date = None
+
+        new_task.importance = json['fields']['customfield_19904']['value']\
```

Comment:
Can you make `'customfield_19904'`, `'customfield_13405'`, and `'customfield_19905'` into class fields with appropriate names that are referenced in all the locations that these special fields are used?